dash_docs/chapters/app_lifecycle/index.py

```diff
@@ -0,0 +1,27 @@
+import dash_core_components as dcc
+import dash_html_components as html
+from dash_docs import tools
+from dash_docs import reusable_components as rc
+
+examples = tools.load_examples(__file__)
+
+layout = html.Div(children=[
+    rc.Markdown('''# Dash App Life Cycle
```

Comment:
```suggestion
    rc.Markdown('''
    # Dash App Life Cycle
```
To ensure our automatic dedenting of markdown works correctly